docker/docker_client.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/containers/image/v5/pkg/tlsclientconfig"
 	"github.com/containers/image/v5/types"
 	clientLib "github.com/docker/distribution/registry/client"
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
Please use `c/storage/pkg/homedir`. It's not yet released though but merged.

Reply:
Updated